Ghidra/Processors/8048/data/languages/8048.slaspec

```diff
@@ -0,0 +1,349 @@
+# sleigh specification file for Intel 8048
+
+
+# Do not take BS into account when decompiling
+@define SINGLE_REGISTER_BANK
```

Comment:
The Sleigh editor needs to have at least an empty string for the value.

# Do not take BS into account when decompiling
@define SINGLE_REGISTER_BANK ""

# Treat R0-R7 as not memory mapped (implies SINGLE_REGISTER_BANK)
@define INTERNAL_REGISTERS ""